### ISSUE TITLE
declaration different in cpp and h file of qstrncmp

### ISSUE THREAD

**Issue by maddin200** [doxygen/doxygen#8247]
```
/qtools/qcstring.cpp line 487:
char *qstrncpy( char *dst, const char *src, uint len )

// qcsting.h line 61:
char * qstrncpy(char *src, const char *dst, uint len);
```

**Comment by albert-github** [comment]
A bit very cryptic issue report, but the difference is a bit confusing. The right syntax is , of course:
```
char *qstrncpy( char *dst, const char *src, uint len )
```

I've just pushed a proposed patch, pull request #8248

**Comment by maddin200** [comment]
ok just naming

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by maddin200** [comment]
very good

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
